Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -154,7 +154,7 @@ namespace AzToolsFramework::Prefab
             return AZ::Failure(AZStd::string("Prefab Focus Handler: Invalid index on FocusOnPathIndex."));
         }
 
-        InstanceOptionalReference focusedInstance = GetReferenceFromContainerEntityId(m_instanceFocusHierarchy[index]);
+        InstanceOptionalReference focusedInstance = GetInstanceReferenceFromRootAliasPath(m_instanceFocusHierarchy[index]);
 
         return FocusOnOwningPrefab(focusedInstance->get().GetContainerEntityId());
```

Comment:
question: Could `InstanceOptionalReference` be invalid/null? (just made me think as it's called 'optional' 😛). Could the `->get()` call cause problems?